tests/lib/rules/no-old-shims.js

```diff
@@ -56,7 +56,7 @@ eslintTester.run('no-old-shims', rule, {
     },
     {
       code: "import Sortable from 'ember-controllers/sortable';",
-      output: "import Sortable from 'ember-controllers/sortable';",
+      output: "import Sortable from 'ember-controllers/sortable';", // eslint-disable-line eslint-plugin/prefer-output-null
```

Comment:
why not `null` in this case?

Reply:
This rule seems to erroneously report that there's an autofix available even though the output is the same as the input. So it doesn't allow me to use `null` here.